src/components/SingleArticle.jsx

```diff
@@ -51,7 +66,9 @@ function SingleArticle() {
       <p>Votes: {singleArticle.votes + voteChange}</p>
       <button onClick={() => handleVote(1)}>Upvote</button>
       <button onClick={() => handleVote(-1)}>Downvote</button>
-      <Comments article_id={article_id} singleArticle={singleArticle}/>
+      {commentSuccessMessage && <p className="success-message">{commentSuccessMessage}</p>}
+      <AddComment article_id={article_id} onCommentAdded={handleCommentAdded} />
+      <Comments comments={comments} singleArticle={singleArticle}/>
```

Comment:
Please try not to change the names of variables :) onCommentAdded===handleCommentAdded